lib/fastlane/plugin/clean_testflight_testers/actions/clean_testflight_testers_action.rb

```diff
@@ -4,39 +4,44 @@ class CleanTestflightTestersAction < Action
       def self.run(params)
         require 'spaceship'
 
-        UI.message("Login to iTunes Connect (#{params[:username]})")
-        Spaceship::Tunes.login(params[:username])
+        app_identifier = params[:app_identifier]
+        username = params[:username]
+
+        UI.message("Login to iTunes Connect (#{username})")
+        Spaceship::Tunes.login(username)
         Spaceship::Tunes.select_team
         UI.message("Login successful")
 
         UI.message("Fetching all TestFlight testers, this might take a few minutes, depending on the number of testers")
 
         # Convert from bundle identifier to app ID
-        spaceship_app ||= Spaceship::Application.find(params[:app_identifier])
-        UI.user_error!("Couldn't find app '#{params[:app_identifier]}' on the account of '#{params[:username]}' on iTunes Connect") unless spaceship_app
-        app_id = spaceship_app.apple_id
+        spaceship_app ||= Spaceship::ConnectAPI::App.find(app_identifier)
+        UI.user_error!("Couldn't find app '#{app_identifier}' on the account of '#{username}' on iTunes Connect") unless spaceship_app
 
-        all_testers = Spaceship::TestFlight::Tester.all(app_id: app_id)
+        all_testers = spaceship_app.get_beta_testers(includes: "betaTesterMetrics", limit: 200)
         counter = 0
 
         all_testers.each do |current_tester|
-          days_since_status_change = (Time.now - current_tester.status_mod_time) / 60.0 / 60.0 / 24.0
+          tester_metrics = current_tester.beta_tester_metrics.first
+
+          time = Time.parse(tester_metrics.last_modified_date)
+          days_since_status_change = (Time.now - time) / 60.0 / 60.0 / 24.0
 
-          if current_tester.status == "invited"
+          if tester_metrics.beta_tester_state == "INVITED"
```

Comment:
I pushed a PR to _Fastlane_ (https://github.com/fastlane/fastlane/pull/15126) to add support for the `INVITED` state.